src/components/bar-plot/NewBarPlot.js

```diff
@@ -19,7 +19,7 @@ import {
 import GeneralChartComponents from '../general-chart-components/GeneralChartComponents';
```

Comment:
Some plots had it passed through and some didn't. This PR updates the plots that didn't.